internal/security/secretstoreclient/common.go

```diff
@@ -26,6 +26,10 @@ import (
 	"net/url"
 )
 
+// noAuthToken need for environment override since compose doesn't allow setting
+// environment variable to empty string
+const noAuthToken = "none"
```

Comment:
I appreciate what you are doing here but there is no real-life scenario where  this is allowed to be be an empty string, if that helps simplify anything.

Reply:
Yes it does help. I'll ping you on slack.

Reply:
Per our call I will remove setting of AuthToken in the non-docker config which removes the need for this constant for override.